news/2022-08-04-wiki-contributors-wanted.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Wiki contributors wanted!"
+date: 2022-08-04 10:00:00 +0000
+---
+
+Lore maestros! Technical aficionados! Information brokers! We seek any kind of knowledge to share in the wiki, and to translate it for every corner of the community.
+
+![](/wiki/shared/news/banners/osu-wiki.jpg)
+
+The [osu! wiki](/wiki/Main_Page) has seen a lot of reconstruction and development over the years. However, while the to-do list grows larger with new ideas and things to expand on, the amount of contributors has remained very small, which has impeded any rapid progress. We are therefore looking for more people to help out!
+
+If you're knowledgeable on certain aspects of the community, whether it be gameplay, mapping, modding, hitsounding, storyboarding or trivia, or simply want to exercise your writing skills, you're more than welcome to participate! No need for Ph.D.'s — any level of contribution is appreciated!
+
+The osu! community reaches every part of the world, which means translations are necessary to keep the knowledge base useful for everyone. By a quick look in the [list of outdated articles and missing translations](https://clayton.cc/osu-wiki-status), the wiki only has French and Indonesian translations almost completely up to date, while every other language has fallen far short over the years. Anyone is encouraged to take part in any amount!
+
+But where does one start? Hop into the [osu!dev Discord server](https://discord.gg/ppy) and check the [contribution guide](/wiki/osu!_wiki/Contribution_guide). The osu! wiki is stored in its entirety in a [GitHub repository](https://github.com/ppy/osu-wiki), where all contribution takes place. Don't worry if you've never used GitHub, just check the linked guide for a quick step-by-step tutorial on the basics and ask any questions in the Discord server.
+
+If you have an idea for improvement, a new article or if you've found some issues, search the [issues page on GitHub](https://github.com/ppy/osu-wiki/issues) to see if someone has already thought of them, and create a [new issue](https://github.com/ppy/osu-wiki/issues/new) if not. If you want to help on an existing task, leave a comment on it to let others know that you're working on it. There is also a [status page](https://clayton.cc/osu-wiki-status/en) which lists all articles that are outdated, in need of clean-up, and article stubs that can be expanded. For more information, see the [article on maintenance of the wiki](/wiki/osu!_wiki/Maintenance#routines).
+
+If you feel like you have an idea but also don't feel like learning GitHub (understandable), just ask any active [osu! wiki maintainer](/wiki/osu!_wiki/Maintenance/List_of_maintainers) through osu! chat, via Discord DM, or in the `#osu-wiki` channel of the [osu!dev Discord server](https://discord.gg/ppy), and they'll make sure your idea is seen through.
```

Comment:
though... i dont know if i would make guarantees someone else is going to do a significant amount of work on your behalf, I feel like that last part should be less certain. in general, this should be conveyed as a less preferable alternative